conditions/README.md

```diff
@@ -0,0 +1,341 @@
+
+| Mapped to (JSON FHIR Condition field)                  | Mapped from (XML HL7 / other source)                                                                                                                                                               |
+|--------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                                     | `LinkSet / id [@root]`                                                                                                                                                                             |
+| meta.profile                                           | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-ProblemHeader-Condition-1"`                                                                                           |
+| extension[0].url                                       | fixed value = `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ProblemSignificance-1"`                                                                                     |
+| extension[0].valueCode                                 | If major is set from the LinkSet then add `"major"` as a value, otherwise add `"minor"`                                                                                                            |
+| extension[index].url                                   | fixed value = `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ActualProblem-1"` <sup>1</sup>                                                                              |
+| extension[index].valueReference.reference              | reference to the mapped resource <sup>1</sup>                                                                                                                                                      |
+| extension[index].url                                   | fixed value = `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-RelatedClinicalContent-1"` <sup>2</sup>                                                                     |
+| extension[index].valueReference.reference              | reference to the mapped resource <sup>2</sup>                                                                                                                                                      |
+| extension[index].url                                   | fixed value = `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-RelatedProblemHeader-1"`  <sup>3</sup>                                                                      |
```

Comment:
I found the duplication of the `url` field confusing.  I wonder if we could combine into a single row somehow?

I'm also confused by the index value.  I know that we have a `LinkSet` we're using, does index match to an ordered position within each Link in that set?  A sentence at the top of the table might help with that.

Reply:
I think I have a better understanding of the code now.

I'm wondering if there is a way to separate out the extensions in a way where it's clear about the different ones which area added.  I don't have a specific idea in my head, but subsectioning them out, so that each note is instead a small intro to the extension, and then the details of the values used are detailed in their own rows perhaps?

Reply:
Yeah, that makes sense. I'll add a new  subsection for them.